docs/src/explanations/interfaces-and-connections.md

```diff
@@ -83,8 +83,63 @@ where <> bulk connects interfaces of opposite gender between sibling modules or
 
 Bulk connections connect leaf ports of the same name to each other. If the names do not match or are missing, Chisel does not generate a connection.
 
-Caution: bulk connections should only be used with **directioned elements** (like IOs), and is not magical (e.g. connecting two wires isn't supported since Chisel can't necessarily figure out the directions automatically [chisel3#603](https://github.com/freechipsproject/chisel3/issues/603)).
+```scala mdoc:silent:reset
+import chisel3._
+
+class SimpleLink extends Bundle {
+  val data = Output(UInt(16.W))
+  val valid = Output(Bool())
+  val ready = Input(Bool())
+}
+class PLink extends SimpleLink {
+  val parity = Output(UInt(5.W))
+}
+class FilterIO extends Bundle {
```

Comment:
since we've already defined SimpleLink, PLink,FilterIO for this example we just can remove these here, and just put the `NotReallyAFilterIO`, and the new `Block`